data-client/src/main/java/org/wikipedia/search/SearchResults.java

```diff
@@ -35,7 +35,7 @@ public SearchResults() {
      * @param suggestion a search suggestion to show to the user: "Did you mean ...?"
      */
     public SearchResults(@NonNull List<MwQueryPage> pages, @NonNull WikiSite wiki,
-                         @Nullable Map<String, String> continuation, @Nullable String suggestion) {
+        @Nullable Map<String, String> continuation, @Nullable String suggestion) {
```

Comment:
I don't see any need for this file to be changed. So, revert the changes to this file. You could use the following commands to revert the changes in this file:

```bash
git fetch origin
git checkout origin/master -- data-client/src/main/java/org/wikipedia/search/SearchResults.java
```

Assuming origin points to this repo.